src/db/tpm.cpp

```diff
@@ -250,67 +252,67 @@ int TPM::headline (rts2image::Image * image, std::ostream & _os)
 
 int TPM::printImage (rts2image::Image * image, WorldCoor * wcs, std::ostream & _os)
 {
```

Comment:
I don't really understand this change, tpm.cpp creates isolated binary...?

Reply:
It is very cryptic even to me. I don't remember touching this file at all, but it was definitely changed, in a commit that does not explain anything about it. So I suppose the whole file may be reverted to original state (but not the whole commit as it has other useful changes)